sdk/core/azure-core-amqp/src/impl/rust_amqp/rust_amqp/azure_core_amqp/README.md

```diff
@@ -6,23 +6,72 @@ NOTE: THIS IS NOT A GENERAL PURPOSE AMQP LIBRARY AND SHOULD NOT BE USED AS SUCH.
 
 This crate is part of a collection of crates: for more information please refer to [https://github.com/azure/azure-sdk-for-rust](https://github.com/azure/azure-sdk-for-rust).
 
-## Example
+## Testing the AMQP Client.
 
-```rust no_run
-use azure_messaging::*;
+The AMQP package is tested using the standard `cargo test` command line:
 
-#[tokio::main]
-async fn main() -> azure_core::Result<()> {
-    let eventhubs_namespace = std::env::var("AZURE_EVENTHUB_NAMESPACE").expect("missing AZURE_EVENTHUB_NAMESPACE");
+```bash
+cargo test --package azure_core_amqp --all-features
+```
+
+Certain functionality of the test requires that the azure-amqp TestAmqpBroker be running at the time of the test. To enable this functionality, first clone the azure-amqp repository to a local directory:
+
+```bash
+cd <Test Working Directory>
+git clone https://github.com/Azure/azure-amqp
+```
+
+Alternately, you can clone to a specific release in the azure-amqp repository:
+
+```
+git clone https://github.com/Azure/azure-amqp.git --branch v2.6.9
+```
 
-    let  client = ProducerClient::new(
-        eventhubs_namespace, credential,
-    )?;
+Next build the TestAmqpBroker binary:
 
-    client.send_event("hello world").await?;
+```bash
+cd azure-amqp/test/TestAmqpBroker
+dotnet publish --self-contained --framework net6.0
```

Comment:
To just run it locally, all these instructions about publishing a self-contained image and CD'ing to the output directory are unnecessary. Just have them run `dotnet run -- amqp://127.0.0.1:25672`. `dotnet publish` it to create packages for other people - not yourself.

Reply:
I'll make this change in the master.